jupyterhub/notebook-images/overlays/additional/kustomization.yaml

```
@@ -1,3 +1,4 @@
+---
 apiVersion: kustomize.config.k8s.io/v1beta1
 kind: Kustomization
 bases:
```

```suggestion
resources:
```
Since `bases` was depreciated in kustomize 2.1.0, and ODH Operator emits warning when applying the notebooks manifests